Fix: Adds a description for Archons (#5274)
endless-sky/endless-sky: data/drak/drak.txt

	turret 55 16 "Drak Distancer"
	explode "nuke explosion" 3
	"final explode" "final explosion large"

> 	description "Very little is know about Archons other than that they significantly exceed the spacefaring prowess of any known species. From external observation, they appear to be a mix of organic and cybernetic elements. They are known to guard noteworthy locations and sapient species or the remains thereof."

Reviewer: ```suggestion
	description "Very little is known about Archons other than that they significantly exceed the spacefaring prowess of any known species. From external observation, they appear to be a mix of organic and cybernetic elements. They are known to guard noteworthy locations and sapient species - or the remains thereof."
```
I like the dash because I think the sentence sounds better with a pause there, but it's personal preferenece.
Also, [sapient](https://www.merriam-webster.com/dictionary/sapient) seems like odd word choice for that line, unless there's another definition I don't know about.

> 	description "Very little is know about Archons other than that they significantly exceed the spacefaring prowess of any known species. From external observation, they appear to be a mix of organic and cybernetic elements. They are known to guard noteworthy locations and seem focused on guarding sapient species or the remains thereof."

Reviewer 2: Repetition.
```suggestion
	description "Very little is know about Archons other than that they significantly exceed the spacefaring prowess of any known species. From external observation, they appear to be a mix of organic and cybernetic elements. They are known to guard noteworthy locations and sapient species or the remains thereof."
```

> 	description "	It is unknown whether Archons are grown, created, or raised. Likewise, while they are identified as 'Drak' by most IFF systems, it is unknown whether or not they are Drak, servants or agents of the Drak, or another thing entirely."

Reviewer 2: ```suggestion
	description "	It is unknown whether Archons are grown, created, or raised. Likewise, while they are identified as 'Drak' by most ships' identification systems, it is unknown whether or not they are Drak, servants or agents of the Drak, or another thing entirely."
```

> 	description "Very little is known about Archons other than that they significantly exceed the spacefaring prowess of any known species. From external observation, they appear to be a mix of organic and cybernetic elements. They are known to guard noteworthy locations and sapient species - or the remains thereof."
> 	description "	It is unknown whether Archons are grown, created, or raised. Likewise, while they are identified as 'Drak' by most ships' identification systems, it is unknown whether or not they are Drak, servants or agents of the Drak, or another thing entirely."

Reviewer 3: I'd go with something much simpler. The description should not simply be a list of those ideas that we've seen scattered throughout the story of the game. (Especially because should this ever become readable in game, we shouldn't basically be spoiling ideas that are mentioned by the story.) See #3070 for the most likely way in which we'd be able to see the description of any ship in game, so the description should fit the setting there (i.e. you're flying around in space with the ship in your system).
```suggestion
	description `The Archon's presence is chilling. The "ship" has an almost organic appearance, but your ship's scanners give you no discernible information on its construction. What exactly it is you do not know, but you can feel that it is powerful.`
```

> 	description "It is unknown whether Archons are grown, created, or raised. Likewise, while they are identified as 'Drak' by most IFF systems, it is unknown whether or not they are Drak, are merely servants or agents of the Drak, or if the labeling is completely erroneous."

Reviewer 2: IFF? International Flavors & Fragrances?
```suggestion
	description "It is unknown whether Archons are grown, created, or raised. Likewise, while they are identified as 'Drak' by most IFF systems, it is unknown whether or not they are Drak, servants or agents of the Drak, or another thing entirely."
```
